src/Config/Stage/Deploy/MergedConfig.php

```diff
@@ -60,26 +63,40 @@ public function __construct(
      * @throws ConfigException If the configuration file can't be read or can't be parsed
      */
     public function get(): array
+    {
+        try {
+            $defaults = $this->schema->getDefaults(StageConfigInterface::STAGE_DEPLOY);
+
+            return $this->mergedConfig($defaults);
+        } catch (FileSystemException $e) {
+            throw new ConfigException($e->getMessage(), Error::DEPLOY_CONFIG_UNABLE_TO_READ_SCHEMA_YAML, $e);
```

Comment:
Same here and in other similar places. I do not see a reason to specify file type. DEPLOY_CONFIG_UNABLE_TO_READ is enough I guess.

Reply:
There are possible errors with two different files, one `.magento.app.yaml` and another one is `schema.yaml` from ece-tools, I thought we should separate these errors somehow